project/main.py

```diff
@@ -109,7 +109,7 @@ def evaluation(experiment: Experiment) -> tuple[str, float | None, dict]:
         # We want to report the training error.
         metrics = {
             **experiment.trainer.logged_metrics,
-            **experiment.trainer.callback_metrics,
```

Comment:
Why this change?

Reply:
fixed in b500fe5a1e47088f6112b207109a48e1bd4b49eb

Reply:
Watch out, this commit also reverted good changes in the top portion of the file!

Reply:
addressed in be21fdbe98b4d1556ae63d6f22c03107992fba95